composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "silverstripe/recipe-form-building",
+    "type": "silverstripe-recipe",
+    "description": "A recipe of modules to help you build forms in SilverStripe",
+    "homepage": "https://www.silverstripe.org",
+    "license": "BSD-3-Clause",
+    "require": {
+        "silverstripe/recipe-plugin": "^1",
+        "silverstripe/recipe-cms": "1.1.x-dev",
+        "silverstripe/segment-field": "2.1.x-dev",
+        "silverstripe/userforms": "5.0.x-dev",
+        "symbiote/silverstripe-queuedjobs": "4.x-dev"
```

Comment:
Since this is only required if you're using gridfieldqueuedexport shall we move this to suggest as well?